README.md

```diff
@@ -72,10 +73,12 @@ yarn add \
   @near-wallet-selector/neth \
   @near-wallet-selector/xdefi \
   @near-wallet-selector/ramper-wallet \
-  @near-wallet-selector/mintbase-wallet \
-  @near-wallet-selector/near-mobile-wallet \
+  @near-wallet-selector/near-mobile-wallet  \
   @near-wallet-selector/bitget-wallet \
-  @near-wallet-selector/okx-wallet
+  @near-wallet-selector/okx-wallet \
+  @near-wallet-selector/mintbase-wallet \
+  @near-wallet-selector/bitte-wallet
+
```

Comment:
extra empty line that need to be removed

Reply:
updated :)